src/driver/verify.hpp

```diff
@@ -37,18 +38,18 @@ void verify_program(const std::string& name,
                     compile_options options     = compile_options{},
                     precision quantize          = precision::fp32,
                     const parameter_map& inputs = {},
-                    double tolerance            = 100);
+                    verify::tolerance tols      = verify::tolerance{});
```

Comment:
Just trivia. The value of tolerance is different in three cases (lines 40,45, 51) . Yet they all have the same replacement default: verify::tolerance{}.

Reply:
yes. It should be fine since the newer default tolerance value is higher compared to what previous value of 100 would have calcualted. 

Default values need to be worked out in a better way.

Reply:
I hope there is a way to dump the tolerances via CLI. Thanks.

Reply:
Made some changes. `driver verify --help` prints all options with defaults.